src/frontend/efiling-frontend/src/modules/userDetails.js

```diff
@@ -1,4 +1,5 @@
 const userDetails = {
```

Comment:
maybe a mock data folder or mock in the name

Reply:
yes will address in subsequent task, thanks!